libraries/chain/database.cpp

```diff
@@ -2283,6 +2285,23 @@ namespace golos { namespace chain {
                             unclaimed_rewards -= claim;
 
                             const auto &voter = get(itr->voter);
+
+                            if (has_hardfork(STEEMIT_HARDFORK_0_19__756)) {
+                                auto vdo_itr = vdo_idx.lower_bound(voter.name);
+                                for (; vdo_itr != vdo_idx.end() && vdo_itr->delegatee == voter.name; ++vdo_itr) {
+                                    if (vdo_itr->interest_rate == 0) {
+                                        continue;
+                                    }
+
+                                    auto delegator_claim = (claim * vdo_itr->interest_rate) / STEEMIT_100_PERCENT;
+
+                                    const auto& delegator = get_account(vdo_itr->delegator);
+                                    create_vesting(delegator, asset(delegator_claim, STEEM_SYMBOL));
```

Comment:
You send claim directly to the delegator. Why? You should add it to delegated vesting.